EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -0,0 +1,390 @@
+```math
+\begin{align}
+Z_0 &\sim \mathcal{N}(0,1),\\
+\sigma^2_Z &\sim \text{HalfNormal}(0.01).
+\end{align}
+```
+"
+
+# ╔═╡ 56ae496b-0094-460b-89cb-526627991717
+rwp = EpiAware.RandomWalk(Normal(),
+    truncated(Normal(0.0, 0.02), 0.0, Inf))
+
+# ╔═╡ 767beffd-1ef5-4e6c-9ac6-edb52e60fb44
+md"
+## Direct infection `EpiModel`
+
+This is a simple model where the unobserved log-infections are directly generated by the latent process $Z$.
+```math
+\log I_t = \log I_0 + Z_t.
+```
+
+As discussed above, we still ask for a defined generation interval, which can be used to calculate $\mathcal{R}_t$.
+
+"
+
+# ╔═╡ 9e43cbe3-94de-44fc-a788-b9c7adb34218
+truth_GI = Gamma(2, 5)
+
+# ╔═╡ f067284f-a1a6-44a6-9b79-f8c2de447673
+md"
+The `EpiData` constructor performs double interval censoring to convert our _continuous_ estimate of the generation interval into a discretized version. We also implement right truncation using the keyword `D_gen`.
+"
+
+# ╔═╡ c0662d48-4b54-4b6d-8c91-ddf4b0e3aa43
+model_data = EpiData(truth_GI, D_gen = 10.0)
+
+# ╔═╡ fd72094f-1b95-4d07-a8b0-ef47dc560dfc
+md"
+We can supply a prior for the initial log_infections.
+"
+
+# ╔═╡ 6639e66f-7725-4976-81b2-6472419d1a62
+log_I0_prior = Normal(log(100.0), 1.0)
+
+# ╔═╡ df5e59f8-3185-4bed-9cca-7c266df17cec
+md"
+And construct the `EpiModel`.
+"
+
+# ╔═╡ 6fbdd8e6-2323-4352-9185-1f31a9cf9012
+epi_model = DirectInfections(model_data, log_I0_prior)
+
+# ╔═╡ 5e62a50a-71f4-4902-b1c9-fdf51fe145fa
+md"
+
+
+### Delayed Observations `ObservationModel`
+
+The observation model is a negative binomial distribution with mean `μ` and cluster factor `1 / r`. Delays are implemented
+as the action of a sparse kernel on the infections $I(t)$.
+
+```math
+\begin{align}
+y_t &\sim \text{NegBinomial}(\mu = \sum_{s\geq 0} K[t, t-s] I(s), r), \\
+1 / r &\sim \text{Gamma}(3, 0.05/3).
+\end{align}
+```
+"
+
+# ╔═╡ e813d547-6100-4c43-b84c-8cebe306bda8
+md"
+We also set up the inference to occur over 100 days.
+"
+
+# ╔═╡ c7580ae6-0db5-448e-8b20-4dd6fcdb1ae0
+time_horizon = 100
+
+# ╔═╡ 0aa3fcbd-0831-45b8-9a2c-7ffbabf5895f
+md"
+We choose a simple observation model where infections are observed 0, 1, 2, 3 days later with equal probability.
+"
+
+# ╔═╡ 448669bc-99f4-4823-b15e-fcc9040ba31b
+obs_model = EpiAware.DelayObservations(
+    fill(0.25, 4),
+    time_horizon,
+    truncated(Gamma(5, 0.05 / 5), 1e-3, 1.0)
+)
+
+# ╔═╡ e49713e8-4840-4083-8e3f-fc52d791be7b
+md"
+## Generate cases from the `EpiAware` model
+
+Having chosen an `EpiModel`, `LatentModel` and `ObservationModel`, we can implement the model as a [`Turing`](https://turinglang.org/dev/) model using  `make_epi_aware`.
+
+By giving `missing` to the first argument, we indicate that case data will be _generated_ from the model rather than treated as fixed.
+"
+
+# ╔═╡ abeff860-58c3-4644-9325-66ffd4446b6d
+full_epi_aware_mdl = make_epi_aware(missing, time_horizon;
+    epi_model = epi_model,
+    latent_model_model = rwp,
```

Comment:
```suggestion
    latent_model = rwp,
```

Reply:
That should change when I merge in latest.

Reply:
It looks like it hasn't

Reply:
Looks ok [here](https://github.com/CDCgov/Rt-without-renewal/blob/add-doc-examples/EpiAware/src/models.jl)?

Reply:
That isn't the part of the code we are looking at though. Updating the code locally can you run this example? Because I think it doesn't work due to line 189 being out of date (i.e my suggestion)

Reply:
Yep that was an error, I read the link too quickly!

Reply:
I've fixed this :-)